index.html

```diff
@@ -1,26 +1,13 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <style>
-      #navbar{
-        position: fixed
-        
-      }
-
-  @media (max-width: 600px) {
-  .facet_sidebar {
-    display: none;
-  }
-}
-    </style>
+<head>
     <meta charset="UTF-8">
     <title>Fork Me! FCC: Test Suite Template</title>
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="stylesheet" media="(max-width: 800px)" href="example.css" />
-  </head>
+    <link rel="stylesheet" href="css/styles.css">
+</head>
 
-  <body>
-    we are a microverse programer very bad 
+<body>
+    We are a Microverse programer - Thechnical Documentation Page (Project) -ivmvh
```

Comment:
This text must be enclosed in some class of tag depending of his semantic meaning.  If is a Title could be <p> y is a title use h1,h2 or h3.
https://www.w3schools.com/html/html5_semantic_elements.asp